dev-python/uri_template/uri_template-1.1.0.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2022 Gentoo Authors
```

Comment:
If it is a new ebuild (as it looks to me), it should be copytight starting from this year, meaning just:
```suggestion
# Copyright 2022 Gentoo Authors
```